tests/kernel/run_ltp.pm

```
@@ -255,6 +255,9 @@ sub run {
         @tests = $self->parse_runfile($cmd_file, $cmd_pattern, $cmd_exclude);
     }
 
+    # some tests fail if dhcp server (started by wickedd) is running
+    script_run('systemctl -q is-active wickedd && { service wickedd stop; export RUN_LTP_START=wickedd; }');
```

Ok, with `script_run()` I can ignore errors and simplify it.